mantidproject/mantid
# Fix documentation for IntegratePeaksMDHKL

Description of work.
Added lines at end of rst file
**To test:**
build documentation and check that IntegratePeaksMDHKL is in
file:///~/mantid/release/docs/html/algorithms/index.html



Fixes #18037



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
